Formula/sqlcmd.rb

```diff
@@ -0,0 +1,22 @@
+class Sqlcmd < Formula
+  desc "Microsoft SQL Server command-line interface"
+  homepage "https://github.com/microsoft/go-sqlcmd"
+  url "https://github.com/microsoft/go-sqlcmd.git",
+  tag:      "v0.8.1",
+  revision: "4e0b95ce49b8164c6496ca1b7a85fa57734fef4c"
```

Comment:
We don't need a Git clone here.
```suggestion
  url "https://github.com/microsoft/go-sqlcmd/archive/refs/tags/v0.9.1.tar.gz"
  sha256 "35a2d165ae6e1f39033ef7ab9ef3b0b205dc1b6b5583912a082297e93ad1c8ab"
```
But, if we must use one for some reason, align the arguments to `url`:
```suggestion
  url "https://github.com/microsoft/go-sqlcmd.git",
      tag:      "v0.8.1",
      revision: "4e0b95ce49b8164c6496ca1b7a85fa57734fef4c"
```